org/ystia/alien/docker/types.yaml

```
@@ -0,0 +1,58 @@
+tosca_definitions_version: alien_dsl_2_0_0
+#
+# Copyright 2020 Bull S.A.S. Atos Technologies - Bull, Rue Jean Jaures, B.P.68, 78340, Les Clayes-sous-Bois, France.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+metadata:
+  template_name: org.ystia.alien.docker
+  template_version: 1.0.0-SNAPSHOT
+  template_author: yorc
+
+imports:
+  - tosca-normative-types:1.0.0-ALIEN20
+  - docker-types:2.2.0-SM10
+
+description: Alien4cloud server that can be deployed to K8S
+
+repositories:
+  docker:
+    url: https://hub.docker.com/
+    type: a4c_ignore
+
+node_types:
+  org.ystia.alien.docker.nodes.Alien:
+    derived_from: tosca.nodes.Container.Application.DockerContainer
+    capabilities:
+      alien_console:
+        type: org.ystia.alien.docker.capabilities.AlienUI
```

Is the capability name rightly chosen ? 
Maybe we should just refer to the capability type because there is only one
```suggestion
    capability: org.ystia.alien.docker.capabilities.AlienUI:
```

---

Maybe a capability name its not necessary here....